packages/@react-native-windows/telemetry/src/CodedError.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ * @format
+ */
+
+export const CodedErrors = {
```

Comment:
FWIW I'm not sure if we've used semantic error codes for anything internally. We could potentially simplify by not making proc exit code guarantees, since not all tools, or all error conditions will anyways